lib/index.iife.js

```diff
@@ -19,6 +19,7 @@ var VueDemi = (function (VueDemi, Vue, VueCompositionAPI) {
     VueDemi.Vue2 = Vue
     VueDemi.version = Vue.version
     VueDemi.warn = Vue.util.warn
+    VueDime.hasInjectionContext = () => !!VueDemi.getCurrentInstance()
```

Comment:
I think there is a typo in your variable, repeated on the second line change

Reply:
Thanks! I created https://github.com/vueuse/vue-demi/pull/227. In this kind of cases, it's even better if you can directly create the PR @noook 😄